src/app/issue/dashboard/card-design/card-design.component.html

```diff
@@ -0,0 +1,53 @@
+<div class="card-wrapper">
+  <div
+    class="card-wrapper__card"
+    *ngFor="let issue of listOfDisplayIssues"
+    (click)="routeToIssue(issue.id)"
+  >
+    <div class="card-wrapper__card__row card-wrapper__card__row--1">
+      <h3>{{ issue.issueDetail.name }}</h3>
+      <app-priority [priority]="issue.issueDetail.priority"></app-priority>
+      <button nz-button nzType="primary" disabled>
+        <i nz-icon nzType="play-circle" nzTheme="outline"></i>
+      </button>
+    </div>
+    <div class="card-wrapper__card__row card-wrapper__card__row--2">
+      <app-progress-bar
+        [progress]="issue.issueDetail.progress"
+        [expectedTime]="issue.issueDetail.expectedTime"
+        [timeSheets]="issue.timeSheets"
+      ></app-progress-bar>
+      <p>
+        <b>Deadline: </b>
+        {{
+          issue.issueDetail.endDate
+            ? (issue.issueDetail.endDate | date: "dd.MM.yyyy")
+            : "---"
+        }}
+      </p>
+    </div>
+    <div class="card-wrapper__card__row card-wrapper__card__row--3">
+      <p>
+        <b>Projekt: </b> {{ issue.project.name ? issue.project.name : "---" }}
+      </p>
+      <p><b>Status: </b> {{ issue.state.name ? issue.state.name : "---" }}</p>
+    </div>
+    <h3 *ngIf="issue.issueDetail.requirements.length > 0">Requirements:</h3>
+    <div>
+      <nz-list nzBordered *ngIf="issue.issueDetail.requirements.length > 0">
+        <nz-list-item
+          *ngFor="
+            let requirement of issue.issueDetail.requirements | slice: 0:3
```

Comment:
Cool wusste garnicht das es die Pipe gibt. Hat es einen Grund wieso du nur die ersten 4 nehmen möchtest? (wenn ich das richtig verstehe)

Reply:
Wollte ich dich gestern auch noch fragen habs aber vergessen. Mein Gedanke war im ersten Moment, dass es ansonsten evtl zu viele werden könnten und es dann etwas unübersichtlich werden könnte.
Aber bin mir nicht mehr so sicher, ob das wirklich sinn macht.

Reply:
Ich glaube alle anzuzeigen, aber das in ein Scroll zu packen, wäre perfekt. Hab auf die schnelle das hier gefunden: https://ng.ant.design/components/affix/en#components-affix-demo-target

Reply:
Ist mit 33b9a4e hinzugefügt